website/content/docs/usage/notebook-sample/index.md

```diff
@@ -1,99 +1,170 @@
 ---
-title: Notebook sample
+title: Kubeflow notebook sample
 weight: 10
 ---
 
-{{< warning >}}
-Incomplete. Show usage of OPA, usage of Egeria, Starting the notebook, Writing code; Also should be a KubeFlow Notebook sample.
-{{</ warning >}}
+This sample shows how to run a Kubeflow notebook with {{< name >}} and demonstrate how polices are seamlessly applied when accessing a dataset.
 
-This guides describes sample usage of {{< name >}} by describing a sample Jupyter notebook application.
-It describes how you, as a data scientist, can use a Jupyter notebook for building a fraud detection model using a 
-data asset you find in the data catalog. 
-
-# Deploy the application
+## Before you begin
+Ensure that you have the following:
+- Kubernetes cluster (this guide was tested with kind v0.10.0 and OpenShift 4.3)
+- Kubectl
+- Kuebflow installed on your cluster (this guide was tested with Kubeflow v1.0.2)
+- S3 Object storage account (e.g., Ceph, Minio, IBM Cloud Object Storage)
 
 {{< tip >}}
-Kubernetes applications can be deployed using any standard deployment tool, from plane `kubectl apply` to [Razee](https://razee.io/).  {{< name >}} only requires a common label.
+You can install Kubeflow on Kind using ```install_kubeflow.sh``` script from {{< name >}} repository as follows:
+```bash
+cd samples/kubeflow/install/kubeflow
+./install_kubeflow.sh
+cd -
+```
 {{</ tip >}}
 
-You have a Kubernetes namespace (OpenShift project) that you can work in, called `fraud-detection`.
-```
-kubectl create namespace fraud-detection
-```
+## About this sample
+In this sample guide you will run a Kubeflow notebook with {{< name >}} and demonstrate that data read polices that are defined in Open Policy Agent (OPA) are seamlessly applied when reading a dataset.
 
-Deploy the Jupyter notebook workload by applying a yaml:
-```yaml
-kind: Deployment
-apiVersion: apps/v1
-metadata:
-  name: notebook-deployment
-  labels:
-    app: my-notebook-app
-spec:
-  replicas: 1
-  selector:
-    matchLabels:
-      app: my-notebook-app
-  template:
-    metadata:
-      labels:
-        app: my-notebook-app
-      annotations:
-        sidecar.istio.io/inject: "true"
-    spec:
-      containers:
-        - name: notebook
-          image: jupyter/datascience-notebook
-          ports:
-            - containerPort: 8888
-              protocol: TCP
----
-kind: Service
-apiVersion: v1
-metadata:
-  name: notebook-service
-  labels:
-    app: my-notebook-app
-spec:
-  ports:
-    - protocol: TCP
-      name: http
-      port: 80
-      targetPort: 8888
-  selector:
-    app: my-notebook-app
-  type: ClusterIP
-```
+In this sample guide you will:
+1. Prepare a dataset to be accessed by the notebook
+1. Register the dataset in ODPi Egeria catalog
+1. Provide {{< name >}} with credentials to the dataset
```

Comment:
You mean register credentials in vault? Should be phrased similar to egeria (catalog vs credential manager)

Reply:
Thanks.